dev-tools/commands/src/upload-track.mjs

```diff
@@ -41,7 +41,7 @@ program.command("upload-track")
   .option("-g, --genre <genre>", "Genre of track (chosen randomly if not specified)")
   .option("-l, --license <license>", "License of track", null)
   .option("-f, --from <from>", "The account to upload track from")
-  .option("-p, --premium-conditions <premium conditions>", "The premium conditions object; sets track as premium")
+  .option("-p, --premium-conditions <premium conditions>", "The premium conditions object; sets track as premium", "")
```

Comment:
this was BS. this being "undefined" as default was making all audius-cmd upload tracks register as premium and make various endpoints not return tracks at all... 😭